src/components/global/Footer.jsx

```diff
@@ -1,8 +1,195 @@
+    icon: <FaGithub size={45} />
+  },
+  {
+    href: 'https://www.linkedin.com/company/code-pdx/',
+    icon: <FaLinkedin size={45} />
+  },
+  {
+    href: 'https://www.meetup.com/Code-for-PDX/',
+    icon: <FaMeetup size={45} />
+  },
+  {
+    href: 'https://discord.gg/x6b573et',
+    icon: <FaDiscord size={45} />
+  }
+];
+
+// renders socialMediaLinks for placement in stack below
+const renderSocialLinks = socialMediaLinks.map(({ href, icon }) => (
+  <Link
+    key={href}
+    href={href}
+    target="_blank"
+    rel="noopener"
+    color="#000"
+    sx={{
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      '&:hover': {
+        color: 'secondary.main'
+      }
+    }}
+  >
+    {icon}
+  </Link>
+));
+
+const socialBlobStyle = {
+  backgroundImage: 'url(/assets/socialsBlob.svg)',
+  backgroundSize: 'cover',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'visible',
+  minWidth: '55%',
+  minHeight: '200px'
+};
+
+const logoBlobStyle = {
+  backgroundImage: 'url(/assets/logoBlob.svg)',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  overflow: 'visible'
+};
+
 const Footer = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+  const theme = useTheme();
+
+  // maps legalLinks used for placement in stack below
+  const renderLegalLinks = legalLinks.map((link, index) => (
+    <Box key={link.title + index}>
+      <Typography
+        key={link.title}
+        variant="body2"
+        color="tertiary.main"
+        mr={'10px'}
+        sx={{
+          display: 'flex',
+          justifyContent: 'flex-end',
+          [theme.breakpoints.down('sm')]: {
+            p: '20px'
+          }
+        }}
+      >
+        {link.text ?? null}
+        <Link
+          href={link.href}
+          underline="none"
+          color="#000"
+          ml={link.ml ?? null}
+          sx={{
+            '&:hover': {
+              color: 'primary.main'
+            }
+          }}
+        >
+          {link.title}
+        </Link>
+      </Typography>
+    </Box>
+  ));
+
+  return (
+    // this box contains the entire footer
+    <Box
+      component="footer"
+      sx={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        position: 'fixed',
+        bottom: 0,
+        width: '100%',
+        padding: '10 40',
+
+        [theme.breakpoints.down('sm')]: {
+          flexDirection: 'column'
+        }
+      }}
+    >
+      {/* this stack holds logo, blob and brand text */}
+      <Stack direction="row" spacing={1} alignItems="center" justifyContent={'space-between'}>
+        <Box sx={{ logoBlobStyle }}>
+          <Box
+            sx={{
+              marginRight: 2,
+              ml: '10px',
+              [theme.breakpoints.down('sm')]: {
+                p: '40px'
```

Comment:
In this current version, I think the mobile version doesn't even need this padding. It looks nice to me without it. Maybe it's still a WIP though.

Reply:
Agreed, not wanted - delete this.   But in general, more can be done for mobile view.